src/ffi.rs

```diff
@@ -131,6 +136,26 @@ pub extern fn quiche_config_set_initial_max_streams_bidi(config: &mut Config, v:
     config.set_initial_max_streams_bidi(v);
 }
 
+#[no_mangle]
+pub extern fn quiche_config_set_initial_max_streams_uni(config: &mut Config, v: u64) {
+    config.set_initial_max_streams_uni(v);
+}
+
+#[no_mangle]
+pub extern fn set_ack_delay_exponent(config: &mut Config, v: u64) {
```

Comment:
Shouldn't these be prefixed with `quiche_config_` for consistency as well?

Reply:
Yes thanks. Fixed in later pushes